### ISSUE TITLE
do_op fails with test ./a.out "9828" ""234" 

### ISSUE THREAD

**Issue by m1gu3l-droid** [JCluzet/42_EXAM#61]
I got this error at do_op function

.system/auto_correc_program.sh: line 62: printf: `"': invalid format character

traces gave me a test that has an extra ' " ' : ./a.out "9828" ""234"

**Comment by JCluzet** [comment]
Hi @izzypt 👋
Which exercise makes this error on do_op? 
The program should normally return an \n if the number of arguments is different from 3.

**Comment by izzypt** [comment]
@JCluzet Hi, from what I remember the tester for the exercise doesnt seem to be working properly.

And above I return a new line but the expected return is a 0 ??

**Comment by JCluzet** [comment]
Yes indeed it is not normal... I confirm you that it should not return 0 but a `\n`, however I do not see anything in the testers who do that... 
If you manage to remake the problem today, let me know where and with what code

**Comment by izzypt** [comment]
I was making some tests today and this is the code I ran on the same exercise this time :
```
#include <unistd.h>
#include <stdio.h>
#include <stdlib.h>

void	do_op(int num1, char *operation, int num2)
{
	int	res;

	if (operation[0] == '+')
		res = num1 + num2;
	if (operation[0] == '-')
		res = num1 - num2;
	if (operation[0] == '*')
		res = num1 * num2;
	if (operation[0] == '/')
		res = num1 / num2;
	printf("%d\n", res);
}

int	main(int argc, char **argv)
{
	if (argc == 4)
		do_op(atoi(argv[1]), argv[2], atoi(argv[3]));
	else
		write(1, "\n", 1);
}
```

It will fail me saying :

----------------8<-------------[ START TEST 
        💻 TEST
./a.out 9828234
        🔎 YOUR OUTPUT:
32527$
        🗝 EXPECTED OUTPUT:
0$
----------------8<------------- END TEST ]
